point_pose[] points_pose